app/models/policy.rb

```diff
@@ -20,4 +31,8 @@ def validate_schema_specification
     return if specification.valid?
     specification.errors[:base].each { |error| errors.add(:schema, error) }
   end
+
+  def set_identifier
+    self.identifier = "#{name}-#{version}"
```

Comment:
This looks safer to me:
```suggestion
    self.identifier ||= "#{name}-#{version}"
```

Reply:
Btw, will it work the search with spaces? https://github.com/3scale/porta/pull/673#discussion_r262474234

Reply:
It should always follow with name and version.